test/jasmine/tests/calcdata_test.js

```diff
@@ -30,7 +30,7 @@ describe('calculated data and points', function() {
         });
     });
 
-    xdescribe('category ordering', function() {
+    describe('category ordering', function() {
```

Comment:
@etpinard is there a recommended script that check pretty much everything? Should I ensure that each commit is free of lint, fdescribe etc. issues? I prefer granular commits but some processes (jasmine and image tests) take significant time and even lint/syntax is something to keep in mind. If commits must be 'clean' in master e.g. to aid `git bisect`, I can squash commits e.g. after a PR review or we could try the new github function which squashes on merge (https://github.com/blog/2141-squash-your-commits)?

Depending on the answer, may I use an existing script e.g. `npm test` that execute multiple aspects, or even make a new one with lint?